src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -487,7 +487,7 @@
       <AdditionalFiles Include="$(SonarProjectOutFolderFilePath)" />
     </ItemGroup>
 
-    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) Analysis configured successfully" />
+    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) Analysis configured successfully with $(SonarProjectOutFolderFilePath)." />
```

Comment:
is there no test for this?

Reply:
🤦‍♂️ And there's a copy/paste issue in there (that I remember fixing already, but not sure why it's not on the PR)